client/app/views/menu.coffee

```diff
@@ -70,10 +70,31 @@ module.exports = class MenuView extends ViewCollection
     toggleDropdown: ->
         @$('#menuitems').toggleClass 'visible'
 
-
+    onCalendarMultipleRemove: ->
+        message = t 'confirm delete selected calendars'
+        if confirm(message)
+            @startSpinner()
+            $('.calendar-actions:checked').each ->
+                calendarName = @value
+                tag = app.tags.getByName calendarName
+                app.calendars.remove calendarName, =>
+                    @stopSpinner()
+        # remove additional menu if only 1 calendar is left
+        if $('#menu-items .calendar-name').length < 2
+            $('#multiple-actions').addClass 'hidden'
     showLoading: ->
         @$('.spinner').show()
+    onCalendarMultipleExport: ->
+        calendars = []
+        $('.calendar-actions:checked').each ->
+            calendars.push(@value)
+        calendars = JSON.stringify calendars
+        window.location = "exportzip/#{calendars}"
```

Comment:
Could you use a post route and send calendars as parameters instead of changing the app location?